### DISCUSSION TITLE
Review for Edge Paths v2.2

### DISCUSSION THREAD

**Discussion by shirshak55** [webdriverio/webdriverio#6299] [General]
Hi,

I am one of the maintainer of edge-paths. Currently I want to release v2.2 which includes linux support. As webdriverio uses it it would be helpful if someone from team can review the PR especially new function name.  If team is busy feel free to close this discussion :)

Every changes are mentioned on the PR.

https://github.com/shirshak55/edge-paths/pull/6

Thank You  :)

**Comment by christian-bromann** [comment]
@shirshak55 thanks for notifying. There doesn't seem to be any breaking changes, so 👍  from our side.
